### PR TITLE
PR: Add a message informing the removal of the Python console

### LINKED ISSUE #4172: Add a message to the Python console about its removal in Spyder 3.2
@dalthviz, the message should be added before the first prompt (i.e. `>>>`) and should say

> NOTE: The Python console is going to be removed in Spyder 3.2. Please start to migrate your work to the IPython console instead.

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #4172

### CODE REVIEW

File: spyder/plugins/externalconsole.py [hunk 1]
```diff
@@ -715,6 +715,11 @@ def process_started(self, shell_id):
             self.help.set_shell(shell.shell)
         if self.variableexplorer is not None:
             self.variableexplorer.add_shellwidget(shell)
+        shell.shell.write("NOTE: The Python console is going to "
+                             "be removed in Spyder 3.2. Please start "
```
Comment (goanpeca) [inline spyder/plugins/externalconsole.py]: What happened to the indendation?

**Comment by ccordoba12** [comment]
Could you post a screenshot?

File: spyder/plugins/externalconsole.py (hunk 1 above)
Comment (dalthviz) [inline spyder/plugins/externalconsole.py]: An excellent question. I will fix this.

**Comment by dalthviz** [comment]
The preview:

imagen

File: spyder/plugins/externalconsole.py [hunk 2]
```diff
@@ -715,6 +715,11 @@ def process_started(self, shell_id):
             self.help.set_shell(shell.shell)
         if self.variableexplorer is not None:
             self.variableexplorer.add_shellwidget(shell)
+        shell.shell.write("NOTE: The Python console is going to "
+                          "be removed in Spyder 3.2. Please start "
```
Comment (ccordoba12) [inline spyder/plugins/externalconsole.py]: removed -> REMOVED

File: spyder/plugins/externalconsole.py [hunk 3]
```diff
@@ -715,6 +715,11 @@ def process_started(self, shell_id):
             self.help.set_shell(shell.shell)
         if self.variableexplorer is not None:
             self.variableexplorer.add_shellwidget(shell)
+        shell.shell.write("NOTE: The Python console is going to "
+                          "be removed in Spyder 3.2. Please start "
+                          "to migrate your work to the "
+                          "IPython console instead.\n", prompt=True)
```
Comment (ccordoba12) [inline spyder/plugins/externalconsole.py]: \n -> \n\n

This is needed to separate this message from the banner below.

**Comment by ccordoba12** [comment]
Thanks @dalthviz!

**Comment by ross-h1** [comment]
Nooooo why are you doing this???

IPython is so unhelpful... just basics like 'Up Arrow' not reproducing last line entered....

Also crashes most of the time i use it... so just give up and go back to Python which works fine.

This is NOT an upgrade!!!

**Pull request merged**